### PR TITLE
Sgts page change hosting service part

### PR DESCRIPTION
**Pull request by anbarisker**

### CODE REVIEW

File: collections/_singapore-government-tech-stack/overview/index.html [hunk 1]
```diff
@@ -180,13 +180,39 @@ <h3>Base Layer</h3>
...
+          <div class="col is-3-desktop is-paddingless is-flex">
+          </div>
+          <div class="col is-4-desktop is-paddingless is-felx">
+          </div>
+          <div class="col is-5-desktop is-8-tablet is-paddingless title-col is-flex" >
+            <div class="col is-paddingless"><p class="has-text-weight-bold">WoG Services</p></div>
+           </div>
+        </div>
+        <div class="row is-paddingless is-marginless host-rows">
+          <div class="col is-2-desktop is-paddingless host-box is-felx " >
```
Comment (teochengyong) [inline collections/_singapore-government-tech-stack/overview/index.html:200]:
```suggestion
          <div class="col is-2-desktop is-paddingless host-box is-flex " >
```

File: collections/_singapore-government-tech-stack/overview/index.html [hunk 2]
```diff
@@ -180,13 +180,39 @@ <h3>Base Layer</h3>
...
+      <div id="hosting-layer-header" class="sgds-container is-fluid">
+        <div class="row margin--bottom--none">
+          <div class="col is-6-desktop is-4-tablet">
+            <h3>Hosting</h3>
+          </div>
+        </div>
+        <div class="row is-paddingless is-marginless is-hidden-mobile">
+          <div class="col is-3-desktop is-paddingless is-flex">
+          </div>
+          <div class="col is-4-desktop is-paddingless is-felx">
```
Comment (teochengyong) [inline collections/_singapore-government-tech-stack/overview/index.html:193]: do you need this?

**Review by teochengyong — CHANGES_REQUESTED** [review]
Looks okay otherwise @anbarisker !

File: collections/_singapore-government-tech-stack/overview/index.html (hunk 2 above)
Comment (anbarisker) [reply collections/_singapore-government-tech-stack/overview/index.html]: removed the col div

**Review by teochengyong — APPROVED** [review]
LGTM!

**Pull request merged**